source/_posts/2015-04-03-artsy-engineering-compensation-framework.markdown

```diff
@@ -0,0 +1,89 @@
+
+We have devised 5 levels, much inspired from the excellent [Rent-the-Runway's Engineering Ladder](http://dresscode.renttherunway.com/blog/ladder). You'll find it at the end of this article.
+
+Armed with this list a team lead can have a good 1:1 conversation with a team member about where they are in their career in terms of pure performance, and what areas need be paid attention to in order to grow into the next level. Leads can also compare engineers that don't work on their teams using a consistent vocabulary.
+
+In parallel with working on this ladder, we did a lot of research regarding market compensation for Engineers. We routinely ask candidates questions about their pay and competitive offers, discuss this with  heads of engineering in other companies, use market data from large recruiting firms and pay attention to polls in the various technology organizations. We understand regional differences and have a good idea of the ranges that a company similar to Artsy pays individuals and constantly update my knowledge. If you run a team and would like to compare your numbers to ours, please e-mail <a href='mailto:db@artsy.net'>db@artsy.net</a>.
+
+A significant portion of your compensation comes in Artsy stock options. However, we do not assign dollar value to those or consider them to do compensation math, other than for the case when we offer you a range of possible salaries and a range of possible stock compensation. We often let you choose high equity and low cash, or low equity and high cash, which shifts down or up your target market value by the difference in dollars between your choice and fair market compensation. We determine the amount of stock options at the time of you joining and you may also receive subsequent grants with the goal of always providing you a fair amount of stock appropriate to your ladder at the moment of the grant.
```

Comment:
..._your_ joining...

Also this is a bit vague without examples. I suggest $100,000,000,000.